### PR TITLE
adds unique_id attribute

### PR DESCRIPTION
**Pull request by Cuteminded**

### CODE REVIEW

File: custom_components/youtube/sensor.py [hunk 1]
```diff
@@ -123,6 +128,7 @@ def icon(self):
     def extra_state_attributes(self):
         """Attributes."""
         return {'url': self.url,
+                'friendly_name': self._name,
```
Comment (ludeeus) [inline custom_components/youtube/sensor.py]: There is no need to define this, HA adds that automatically

**Review by ludeeus — CHANGES_REQUESTED** [review]

**Review by pinkywafer — CHANGES_REQUESTED** [review]
Yes, please make the changes @ludeeus requested, them I'm happy for this to be merged.

File: custom_components/youtube/sensor.py (hunk 1 above)
Comment (Cuteminded) [reply custom_components/youtube/sensor.py]: I have removed friendly_name from the Attributes

**Comment by pinkywafer** [comment]
Great.  Thanks @DeadlyKltten

**Pull request merged**